app/src/main/java/com/sopt/now/compose/data/dto/response/ResponseLoginDto.kt

```diff
@@ -0,0 +1,12 @@
+package com.sopt.now.compose.data.dto.response
+
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+@Serializable
+data class ResponseLoginDto(
+    @SerialName("code")
+    val code: Int,
+    @SerialName("message")
+    val message: String,
+)
```

Comment:
DTO들을 잘 선언해주셨네요 👍

이렇게 해도 문제는 없는데 하나의 팁을 더 드리면, Serializable을 사용하면서 data class에 default 값들을 넣어주면 나중에 서버에서 값이 null로 내려오거나 아무런 값도 내려오지 않았을 때 앱이 exception을 던지지 않고 Default 값을 대체해줘요. 그래서 default 값을 넣어주어도 좋은데 이거는 그냥 알아만 두시거나 다음 작업부터 반영해주셔도 될 것 같아요!

default 값을 넣게 되면 아래처럼 될거에요.
```suggestion
@Serializable
data class ResponseLoginDto(
    @SerialName("code")
    val code: Int = -1,
    @SerialName("message")
    val message: String = "",
)
```

Reply:
api 명세서에서 해당 필드를 nullable하지 않다고 해도 이렇게 기본값으로 설정하면 되는건가요!?

Reply:
옙 서버에서 내려주는 데이터는 항상 확신할 수는 없으니까요!
- 서버에서 명세가 바뀌었지만 명세서가 변경되지 않았을 경우
- 특정 오류로 인해 특정 필드가 내려오지 않은 경우

이러한 경우에 필드가 널러블 하지 않은 것과 관계 없이 필드 자체가 내려오지 않을 수 있습니다.
그렇기에 앱에서는 항상 서버에서 데이터를 잘 주지 않았을 경우도 고려해서 DTO를 설계한다면 더욱 안전한 프로젝트의 개발이 가능할 것 입니다ㅎㅎㅎ